src/Shamsheer.Service/DTOs/Users/UserForLoginDto.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Shamsheer.Service.DTOs.Users;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Shamsheer.Service.DTOs.Users;
 
 public class UserForLoginDto
 {
+    [Required]
+    [EmailValidation(ErrorMessage = "Invalid email format!")]
     public string Email { get; set; } = string.Empty;
     public string Password { get; set; } = string.Empty;
```

Comment:
Why did you use string.empty?

Reply:
It was not my work.By the way,I will fix that too.